docs/user-guide/data-structures/data-structures.ipynb

```diff
@@ -161,18 +220,25 @@
     "\n",
     "Scalar variables are distinct from arrays that contain a single value.\n",
     "For example, `sc.scalar(1)` is equivalent to `sc.array(dims=[], values=1)`.\n",
-    "But all of the following are distinct:\n",
+    "But all the following are distinct:\n",
     "\n",
     " - `sc.array(dims=[], values=1)`\n",
     " - `sc.array(dims=['x'], values=[1])`\n",
     " - `sc.array(dims=['x', 'y'], values=[[1]])`\n",
     "\n",
+    "In particular, the first is a scalar while the other two are not; they are arrays with an extent of one.\n",
+    "Using the `value` property with them raises an exception because it requires a truly 0-dimensional variable.\n",
```

Comment:
```suggestion
    "Accessing the `value` property of one of the latter two variables would raise an exception because this property requires a 0-dimensional variable.\n",
```